plugins/org.jkiss.dbeaver.ui.editors.base/src/org/jkiss/dbeaver/ui/editors/text/handlers/ToggleSingleLineCommentHandler.java

```diff
@@ -48,13 +50,18 @@ protected void processAction(ISelectionProvider selectionProvider, ICommentsSupp
                 break;
             }
         }
+        
         for (int lineNum = endLine; lineNum >= startLine; lineNum--) {
             int lineOffset = document.getLineOffset(lineNum);
             int lineLength = document.getLineLength(lineNum);
             if (forceComment) {
                 // Add comment
                 document.replace(lineOffset, 0, singleLineComments[0]);
-                selOffset += singleLineComments[0].length();
+                if(lineNum == startLine) {
+                	selOffset += singleLineComments[0].length();
+                } else {
+                	selLength += singleLineComments[0].length();
+                }
```

Comment:
Please fix the formatting. We don't use tabs for indentation, and there's a missing space after the `if` keyword.